packages/models/src/step/service.ts

```diff
@@ -428,6 +442,11 @@ const createStep = <T>(
   id: string,
   stepType: T extends Step<infer U, infer K> ? K : never,
   state: T extends Step<infer U, infer K> ? U : never,
+  owner?: User,
```

Comment:
I'd rather have last argument like `data` or something like that containing all the remaining data.

```
createStep = (id, stepType, state, data? = {}) => ({id, stepType, state, ...data})
```